### PR TITLE
fixed util/isFunction bug

### PR DESCRIPTION
**Pull request by jphamilton**
This removes the check for lodash/underscore from the util function and adds the missing isFunction function to the shim.

### CODE REVIEW

File: src/jquery.floatThead.js [hunk 1]
```diff
@@ -95,6 +95,10 @@
         return result;
       };
     };
+    that.isFunction = function(obj) {
```
Comment (mkoryak) [inline src/jquery.floatThead.js]: thanks for the PR, but this isn't needed. it should be already handled above with the loop over `isThings` on line 71

Comment (jphamilton) [inline src/jquery.floatThead.js]: I don't understand how. There are several calls to util.isFunction for which I was getting isFunction not defined using the latest release and  with a build I did from the latest code. I am not using underscore or lodash. When I added isFunction, everything started to work.

Comment (mkoryak) [inline src/jquery.floatThead.js]: are you sure you didnt have your own local version of `window._` without that method?

does this code not do what I think it does?

```js
$.each(isThings, function(){
       var name = this;
       that['is' + name] = function(obj) {
         return Object.prototype.toString.call(obj) == '[object ' + name + ']';
       };
     });
```

Comment (jphamilton) [inline src/jquery.floatThead.js]: Agghh, it does work. I am not sure what happened. The fix is just removing the window._

I removed the isFunction part and updated. Sorry for the confusion.

File: dist/jquery.floatThead.js [hunk 2]
```diff
@@ -237,7 +237,7 @@
         w += parseInt($table.css("borderRight"), 10);
       }
       for(var i=0; i < $fthCells.length; i++){
-        w += $fthCells.get(i).offsetWidth;
+        w += getOffsetWidth($fthCells.get(i));
```
Comment (mkoryak) [inline dist/jquery.floatThead.js]: What's going on here?

Comment (mkoryak) [inline dist/jquery.floatThead.js]: Ah I see

**Comment by mkoryak** [comment]
Thanks again for the PR

**Comment by mkoryak** [comment]
wtf, i didnt press the right buttons

**Pull request merged**